docs/src/man/comparisons.md

```diff
@@ -0,0 +1,31 @@
+The table compares the main functions of DataFrames.jl with the R package dplyr (v1) and Stata (>=v8)
+|Operations| DataFrames.jl       | dplyr | Stata|
+|:------------|:------------|:------------|:------------|
+|Reduce multiple values|`combine(df, :x => mean)`|`summarize(df, mean(x))`|`collapse (mean) x =`|
+|Add new columns|`transform(df, :x => mean => :x_mean)`|`mutate(df, x_mean = mean(x))`|`egen x_mean = mean(x)`|
+|Pick columns|`select(df, :x, :y)`|`select(df, x, y)`|`keep x y`|
+|Pick rows |`filter(:x => x -> x >= 1, df)`|`filter(df, x >= 1)`|`keep if x >= 1`|
```

Comment:
I was thinking about it - if you want to show how to filter using a custom defined predicate in Julia then probably it should be shown in R and Stata how to define such a predicate. Otherwise in Julia you should write `>=(1)` as I have suggested earlier I think. Or maybe show both in two consecutive lines?

Reply:
I don't think it's super easy to understand your suggested syntax. You have to know that >=(1) creates a Function which is really not obvious. I changed the predicate to ismissing to sidestep this.

Reply:
But this is exactly the point - we want to teach readers here that there is such an option I think; otherwise we will always have a problem that they do not know this syntax, because we do not show it :smile: .
I would keep `ismissing` example as it is nice

Reply:
Maybe there can be another line below showing both `:x => x -> x > 1` and `:x => >(1)` in the same cell? I agree with @bkamins that it's a good occasion to show people how to do this. If we only show `ismissing`, people won't even now how to do more complex operations.